pyproj/_crs.pyx

```diff
@@ -1599,28 +1599,35 @@ cdef class _CRS(Base):
 
         return None
 
+    def _is_crs_property(self, property_name, property_types, sub_crs_index=0):
+        if self.sub_crs_list:
+            sub_crs = self.sub_crs_list[sub_crs_index]
+            if sub_crs.is_bound:
+                is_property = getattr(sub_crs.source_crs, property_name)
+            else:
+                is_property = getattr(sub_crs, property_name)
+        elif self.is_bound:
+            is_property = getattr(self.source_crs, property_name)
+        else:
+            is_property = self._type in property_types
+        return is_property
+
+
     @property
     def is_geographic(self):
         """
         Returns
         -------
         bool: True if projection in geographic (lon/lat) coordinates.
```

Comment:
the "projection" here is not really correct? (as geographic CRS is not projected?)

Reply:
Good point - I will attempt to clarify.